source/Octopus.Cli/Commands/Releases/PromoteReleaseCommand.cs

```diff
@@ -44,14 +46,20 @@ public async Task Request()
 
             environment = await Repository.Environments.FindByNameOrIdOrFail(FromEnvironmentNameOrId).ConfigureAwait(false);
 
-            var dashboard = await Repository.Dashboards.GetDynamicDashboard(new[] {project.Id}, new[] {environment.Id}).ConfigureAwait(false);
-            var dashboardItem = dashboard.Items.Where(e => e.EnvironmentId == environment.Id && e.ProjectId == project.Id)
-                .OrderByDescending(i => SemanticVersion.Parse(i.ReleaseVersion))
-                .FirstOrDefault();
+            var dashboard = await Repository.Dashboards.GetDynamicDashboard(new[] {project.Id}, new[] {environment.Id}, true).ConfigureAwait(false);
```

Comment:
Wondering if passing true here has the potential to affect the behaviour? If it returns a release that it would not have before, and that happens to have the highest version. I think a conservative approach would be to pass `UseLatestSuccessfulRelease` to the API call as well.

Reply:
woops, yeah `UseLatestSuccessfulRelease` should be passed in instead